legal-api/migrations/versions/441a5e6cf682_.py

```diff
@@ -65,6 +65,9 @@ def upgrade():
             },
             {
                 'identifier':'recordsOffice', 'description':'Records Office'
+            },
+            {
+                'identifier':'custodialOffice', 'description':'Custodial Office'
```

Comment:
Will this work since this version is already migrated? or are you planning to insert directly in DB?

Reply:
yeah, I wanted to keep the codes together, and haven't refactored the code loading yet.
yes, was planning on running a separate patch across environs as I already had to load it into Prod to complete the OPs ticket